movement/io/load_bboxes.py

```diff
@@ -274,10 +274,15 @@ def from_via_tracks_file(
 
     Notes
     -----
-    The bounding boxes IDs specified in the "track" field of the VIA
-    tracks .csv file are mapped to the "individual_name" column of the
-    ``movement`` dataset. The individual names follow the format ``id_<N>``,
-    with N being the bounding box ID.
+    Note that the x,y coordinates in the input VIA tracks .csv file
+    represent the the top-left corner of each bounding box. Instead the
+    corresponding ``movement`` dataset holds in its `position` array the
+    centroid of each bounding box.
+
+    Additionally, the bounding boxes IDs specified in the "track" field of
+    the VIA tracks .csv file are mapped to the "individual_name" column of
+    the ``movement`` dataset. The individual names follow the format
+    ``id_<N>``, with N being the bounding box ID.
```

Comment:
I had missed that before, but "individual_name" column assumes a pandas dataframe. The dataset returned by this funciton is an xarray dataset, so maybe more accurate to say sth like "are mapped to the coordinates of the ``individuals`` dimension in the ``movement`` dataset", or more simply "are mapped to individuals in the ``movement`` dataset". The following sentence starting with "The individual names follow..." is fine as is.